File: app/src/main/java/com/kongpf8848/dmail/bean/OAuthToken.java
package com.kongpf8848.dmail.bean;

import com.kongpf8848.dmail.oauth.DMAccountType;

import java.io.Serializable;

public class OAuthToken implements Serializable {
    public DMAccountType accountType;
    public String access_token;
    public String refresh_token;
    public Long expire_time;

}

File: app/src/main/java/com/kongpf8848/dmail/bean/OAuthToken.kt
package com.kongpf8848.dmail.bean

import com.kongpf8848.dmail.login.DMAccountType
import java.io.Serializable

class OAuthToken : Serializable {
    @JvmField
    var accountType: DMAccountType? = null
    @JvmField
    var access_token: String? = null
    @JvmField
    var refresh_token: String? = null
    @JvmField
    var expire_time: Long? = null
}